Makefile.am

```diff
@@ -6,11 +6,6 @@ SUBDIRS=safe_c_stub lib bin
 joy_api:
 	(doxygen Doxyfile)
 
-cscope:
```

Comment:
Centos7 had issues with this target. Removing for now.

Reply:
I use this target all the time. What was the specific issue?

Reply:
I'm wondering if it's missing cscope from the base image.
I normally save off "find . -type f -follow -name "*.[ch]" -print > cscope.files" and run it as a script.

Reply:
Actually cscope is a new default target for this autotools version. You can run make cscope as before.